index.js

```diff
@@ -16,6 +16,9 @@ function loader (registryOrVersion) {
     windows['minecraft:generic_9x5'] = { type: protocolId++, inventory: { start: 5 * 9, end: 5 * 9 + 35 }, slots: 5 * 9 + 36, craft: -1, requireConfirmation: true }
     windows['minecraft:generic_9x6'] = { type: protocolId++, inventory: { start: 6 * 9, end: 6 * 9 + 35 }, slots: 6 * 9 + 36, craft: -1, requireConfirmation: true }
     windows['minecraft:generic_3x3'] = { type: protocolId++, inventory: { start: 3 * 3, end: 3 * 3 + 35 }, slots: 3 * 3 + 36, craft: -1, requireConfirmation: true }
+    if (registry.supportFeature('chatPacketsUseNbtComponents')) {
```

Comment:
How is this feature related?

Reply:
If it's not I'd say it's better to just check if the current registry version >= 1.20.3 and add feature later than leave with wrong feature here